deprecations.js

```diff
@@ -2,10 +2,8 @@
 
 const warning = require('process-warning')()
 
-const warnName = 'PinoHttpWarning'
-
-warning.create(warnName, 'PINOHTTP_DEP001', 'Use of "autoLogging.ignorePaths" is deprecated since version 7.1.0, use "opts.autoLogging.ignore" instead.')
-
-warning.create(warnName, 'PINOHTTP_DEP002', 'Use of "autoLogging.getPath" is deprecated since version 7.1.0, use "opts.autoLogging.ignore" instead.')
+// Warning example
+// const warnName = 'PinoHttpWarning'
+// warning.create(warnName, 'PINOHTTP_DEP001', 'Use of "autoLogging.ignorePaths" is deprecated since version 7.1.0, use "opts.autoLogging.ignore" instead.')
```

Comment:
Please keep both warnings around so that we know what number we are on. As written, I am quite certain that the next time we add a deprecation it will get tagged as `PINOHTTP_DEP002`.

Reply:
ok, done!